server/utils/constants.js

```diff
@@ -0,0 +1,7 @@
+const EVENTS = {
+  MRD_RECOMMENDATION_STARTED: 'mrdRecommendationStarted',
+}
+
+module.exports = Object.freeze({
```

Comment:
Named exports are more helpful as the IDE will suggest the names as you start typing

Reply:
It's consistent to use a .ts file extension here

Reply:
Fixed